open-metadata-resources/open-metadata-deployment/ansible-egeria/README.md

```diff
@@ -0,0 +1,166 @@
+
+
+### egeria-omag-server-admin
+
+**Administration** is handled by `egeria-omag-server-admin` role. It looks up for specific `admin_omag_*` set of configuration variables and calls server admin REST API endpoints to create/update the configuration documents for each Egeria component. See role specific configuration and variable section for more details.
+
+The sequence of tasks:
+- Configure servers - uses `admin_omag_server_config` dictionary variable;
+- Configure repositories - uses `admin_omag_repo_config` dictionary variable;
+- Enable access services - uses `admin_omag_access_services_config` dictionary variable;
+- Start server instance - uses `admin_omag_server_startup` list.
+
+Each of these sequences depends on the presence of of specific var. If not present or empty the specific configuration phase is skipped.
+
+>Currently the roles are embedded and maintained into this project. Once they get more stable and mature can be also available via ansible-galaxy for other ansible projects using Egeria.
+
+## Configuration and variables
+
+### egeria-omag-server
+
+`atr_artifactory_base` - Source Artifactory url. Default: `https://odpi.jfrog.io/odpi`
+
+`atr_repository` - Source Artifactory repository. Default: `egeria-snapshot-local`
+
+`home_dir` - Target server home directory (installation base). Default: `/opt/odpi/egeria`
+
+`app_host` - Application host address or dns name on the target server.
+
+`app_port` - Application port on the target server.
+
+`app_name` - Application name. Also used for systemd service name.
+
+`app_version`- Version of the egeria core executable (jar).
+
+`app_version_suffix` - Version suffix (helps for testing snapshot versions). <!--#TODO: add more details-->
+
+`app_force_replace` - Forces overwriting the destination artifacts. Useful for development and testing especially when snapshot versions are rebuilt.
+
+`app_extra_libs` - Additional egeria plugin library (i.e. connectors).
+
+`app_configs`-  Location of the application configuration files to be applied on the target server. <!--#TODO: add more details-->
+
+
+### egeria-omag-server-admin
+
+`admin_omag_server_config` - OMAG Server core configuration object. 
+It can be used to define multiple server configurations (different logical servers). See Examples for details.
+
+<!--
+#TODO: add more details on how mode works and possible options
+-->
+
+`admin_omag_repo_config` - OMRS, repositories configuration object. You can define multiple repository configurations for servers. See Examples for details.
+
+`admin_omag_access_services_config` - OMAS, access services configuration object. You can define multiple access services to be enabled per server or simply enable 'all' available. See Examples for details.
+
+`admin_omag_server_startup` - List of services to initialize after configuration is performed. See Examples for details.
+
+
+## Examples
+
+**Example A.** OMAG Server with Asset Catalog OMAS and Local Graph Repository
+
+```
+app_port: 8080
```

Comment:
may want to change these defaults to be 2.2-SNAPSHOT & also update ports to current standards ie 9443 & https